package.json

```diff
@@ -73,8 +73,8 @@
     "@babel/core": "7.18.6",
     "@babel/generator": "7.18.7",
     "@babel/types": "7.18.8",
-    "@commitlint/cli": "7.2.0",
-    "@commitlint/config-conventional": "7.1.2",
+    "@commitlint/cli": "17.0.3",
```

Comment:
ℹ️ 7.2.0 was released 4 years ago. 😱

Reply:
😨  Yikes. Thank you for updating all of this!